app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/multiple_disposals_task_list.scala.html

```diff
@@ -43,6 +44,14 @@
     )
 }
 
+@ytdLiabilityState = @{
```

Comment:
the state should be cannot start at certain times but we don't have all the sections available to do that properly yet